native/packages/react-native-bpk-component-card/src/BpkCard.js

```diff
@@ -83,40 +78,34 @@ const styles = StyleSheet.create({
 
 const BpkCard = (props) => {
   const {
-    padded, children, focused, onPress, style, ...rest
+    padded,
+    children,
+    focused,
+    style: userStyle,
+    ...rest
   } = props;
 
-  const cardBaseStyle = [styles.card, styles.common];
-
-  if (padded) {
-    cardBaseStyle.push(styles.padded);
-  }
-
-  if (focused) {
-    cardBaseStyle.push(styles.focused);
-  }
+  const style = [styles.card];
+  if (padded) { style.push(styles.cardPadded); }
+  if (focused) { style.push(styles.cardFocused); }
+  if (userStyle) { style.push(userStyle); }
 
   return (
     <TouchableHighlight
       accessibilityComponentType="button"
-      onPress={onPress}
-      style={[cardBaseStyle, style]}
       underlayColor={colorGray50}
+      style={style}
       {...rest}
     >
-      <View
-        style={[styles.overflowHidden, styles.common]}
-      >
-        {children}
-      </View>
+      <View>{children}</View>
     </TouchableHighlight>
   );
 };
 
 BpkCard.propTypes = {
   children: PropTypes.node.isRequired,
-  focused: PropTypes.bool,
   onPress: PropTypes.func.isRequired,
+  focused: PropTypes.bool,
```

Comment:
they were in alphabetical order, not anymore 😢 but required are on top, mixed feelings here

Reply:
if you look at the rest of the codebase, `required` are always on top. We should stick to conventions.

Reply:
yes I know, that's why mixed feelings, it's in line with the codebase but not alphabetical anymore 😄